packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.tsx

```diff
@@ -35,7 +35,7 @@ function DefaultValueBooleanField(props: DefaultValueFieldProps<string>) {
       <Select
         {...SelectProps}
         disabled={disabled}
-        value={value}
+        value={value || ''}
```

Comment:
Fix a console.warning because this cannot be `undefined` if it's a controlled component